src/Compiler/CompilerUtils.cpp

```diff
@@ -308,6 +308,26 @@ static void genLLVMBitcode(const mlir::OwningOpRef<ModuleOp> &module,
   llvm::Metadata *identNode[] = {llvm::MDString::get(ctx, OnnxMlirVersion)};
   identMetadata->addOperand(llvm::MDNode::get(ctx, identNode));
 
+  // Set functions to be accesible from DLL on Windows.
+  if (llvm::Triple(getTargetTripleOption()).isOSWindows()) {
```

Comment:
@sstamenova it seems we cannot detect Windows OS from triple. I don't know why it is. I don't have Windows environement, so could you help check what happened here? 

There are several functions that can be called to check different  Windows environments here: https://github.com/llvm/llvm-project/blob/main/llvm/include/llvm/ADT/Triple.h#L561, and I chose the general one.

Reply:
@tungld : What do you mean? `isOSWindows` should return `true` when the triple is targeting Windows, what's the case when you saw it fail?

Reply:
It looks `llvm::Triple(getTargetTripleOption()).isOSWindows()` returned false so that the code inside if was not executed, and numerical tests failed. If I used `if(true)`, the code was executed and numerical tests passed. I suspect that `llvm::Triple(getTargetTripleOption()).isOSWindows()` returned false. I cannot confirm this from Windows CI logs, could you please check on your local environment? Thanks!

Reply:
I switched to use `ifdef __WIN32` instead of triple.